src/ensembl/src/content/app/entity-viewer/gene-view/components/unspliced-transcript/UnsplicedTranscript.tsx

```diff
@@ -199,12 +194,12 @@ const ExonBlock = (props: ExonBlockProps) => {
   const y = -3;
   const height = BLOCK_HEIGHT;
 
-  const exonClasses = classNames(styles.exon, props.className);
+  const exonClasses = props.className;
```

Comment:
sorry @azangru is this missing something here, should it not be:
props.classNames?.exon || undefined;
if i understand this right, the exon class is coming from the css but in your change you are omitting it

Reply:
Line 197 is within a react component called ExonBlock. This component is receiving an optional className property that is just for the exon from UnsplicedTranscript [here](https://github.com/Ensembl/ensembl-client/blob/eb0423055e325364d257966f141dcb0bf0589a46/src/ensembl/src/content/app/entity-viewer/gene-view/components/unspliced-transcript/UnsplicedTranscript.tsx#L77-L82)